bluemira/equilibria/profiles.py

```diff
@@ -620,6 +621,9 @@ def jtor(
             ii, jj = np.nonzero(mask)
             jtorshape *= mask
             pfunc = speedy_pressure_mask(ii, jj, psi_norm, psio, psix, self.shape)
+            if np.isnan(pfunc).any():
+                bluemira_warn("Found NaN in pressure mask, setting NaN's to 0")
+                np.nan_to_num(pfunc, nan=0.0, copy=False)
```

Comment:
Tja, fine, perhaps. If I ever see this in the wild I will probably waste some time looking at it. Please don't remove this warning until then.